bottlecap/src/metrics/enhanced/lambda.rs

```diff
@@ -55,6 +65,9 @@ impl Lambda {
     }
 
     pub fn set_runtime_duration_metric(&self, duration_ms: f64) {
+        if !self.config.enhanced_metrics {
+            return;
```

Comment:
Why those methods are not returning anything, unlike the others above?

Reply:
I agree we should standardize. I think removing the result is the most clean way, because we don't retry/reattempt incrementing metrics if it fails, right?

Reply:
Done